micro-ui/web/micro-ui-internals/packages/react-components/src/molecules/FilterAction.js

```
@@ -1,11 +1,12 @@
 import React from "react";
 import { FilterSvg,FilterIcon,SortSvg } from "../atoms/svgindex";
 import RoundedLabel from "../atoms/RoundedLabel";
+import { COLOR_FILL } from "../atoms/contants";
 
 const FilterAction = ({ text, handleActionClick, ...props }) => (
   <div className="searchAction" onClick={handleActionClick}>
     <RoundedLabel count={props.filterCount}></RoundedLabel>
```

Biome

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)